client/src/main/java/io/pravega/client/control/impl/ModelHelper.java

```diff
@@ -416,10 +418,18 @@ public static final NodeUri decode(final PravegaNodeUri uri) {
         return Controller.ScopeInfo.newBuilder().setScope(scope).build();
     }
 
-    public static final StreamInfo createStreamInfo(final String scope, final String stream) {
+    public static final StreamInfo createStreamInfo(final String scope, final String stream, AccessOperation accessOperation) {
         Exceptions.checkNotNullOrEmpty(scope, "scope");
         Exceptions.checkNotNullOrEmpty(stream, "stream");
-        return StreamInfo.newBuilder().setScope(scope).setStream(stream).build();
+        StreamInfo.Builder builder = StreamInfo.newBuilder().setScope(scope).setStream(stream);
+        if (accessOperation != null) {
+            builder.setRequestedPermission(accessOperation.name());
+        }
```

Comment:
... `null` is equivalent to `NONE`?

Reply:
No, not really. Older clients will not set the access operation, so the server will not receive a requested permission in the input message.

Reply:
Ok, so `null` is for backward compatibility?

Reply:
Yes.